packages/std/src/ibc.rs

```diff
@@ -297,11 +296,7 @@ impl From<IbcChannelOpenMsg> for IbcChannel {
     }
 }
 
-/// Note that this serializes as "null".
-#[cfg(not(feature = "ibc3"))]
-pub type IbcChannelOpenResponse = ();
 /// This serializes either as "null" or a JSON object.
-#[cfg(feature = "ibc3")]
 pub type IbcChannelOpenResponse = Option<Ibc3ChannelOpenResponse>;
```

Comment:
Would be good to document what returning `None`/`null` means semantically here. I don't know to be honest. Could you check the wasmd implementation for that?

Reply:
In wasmd, it is translated to an [empty version string](https://github.com/CosmWasm/wasmd/blob/606289ab56dbe803bc28941a4b4c361b26435d37/x/wasm/keeper/relay.go#L48), and then later replaced with the incoming version (provided by ibc-go? I'm not sure how that works).
So, using `None` here means I don't want to provide a channel version and accept whatever version it is right now. I added this to the doc comment.
Seems a bit confusing to me that this is specified one level above where the version would be specified. We could change this to use `version: Option<String>` and remove the Option from IbcChannelOpenResponse, but wasmd would obviously still have to support both variants.

Reply:
You are very right, this interface is not ideal. But let's keep the top level nullability to remind ourselves that this needs to remain supported in wasmd. We can look at this again in a future version.